packages/@o3r/core/schematics/sdk/rules/update-ts-paths.rule.ts

```diff
@@ -34,6 +34,9 @@ export function updateTsConfig(targetPath: string, projectName: string, scope: s
       `${relativeTargetPath}/dist`,
       `${relativeTargetPath}/src/index`
     ];
+    configWithPath.content.compilerOptions.paths[`${scope ? `@${scope}/` : ''}${projectName}/fixtures`] = [
+      `${relativeTargetPath}/src/fixtures/jest`
```

Comment:
is it ok to add this without knowing if `jest` will be used to run the tests ?

Reply:
For now sdk comes generated directly with 'jest', so the change is fixing the missing path. 
I link again the issue of the alignement of project generation https://github.com/AmadeusITGroup/otter/issues/882